Say Their Names/Source/Controller/Base/BaseViewController.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  BaseViewController.swift
+//  Say Their Names
+//
+//  Created by evilpenguin on 5/31/20.
+//  Copyright © 2020 Franck-Stephane Ndame Mpouli. All rights reserved.
+//
+
+import UIKit
+
+class BaseViewController: UIViewController {
+    public weak var service: Service!
```

Comment:
We can add a new coder initializer that will accept the `Service` as a parameter so we dont need a force unwrapped `Service`
```
init?(coder: NSCoder, service: Service) {
    self.service = service
    super.init(coder: coder)
}
```

Reply:
Agree, but I didn't go the coder approach. Pushed changes.